cmd/generate_csr/main.go

```diff
@@ -0,0 +1,66 @@
+// Copyright 2017 Istio Authors
```

Comment:
The node agent that lives in node, see https://docs.google.com/document/d/1RZnBfj8I5xs8Yi_BPEKBRp0K3UnIJYTDg_31rfTt4j8/edit#heading=h.p0zc37bjhmkg